frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx

```diff
@@ -226,7 +227,7 @@ export const LogDetails: React.FC<Props> = ({
 						kind="secondary"
 						emphasis="low"
 						onClick={(e) => {
-							const url = getLogURL(row)
+							const url = getLogURL(projectId, row)
```

Comment:
Unrelated to this actual issue but we need to attach the project id to the permalink log. 
Permalinking is pretty darn slow (a distinct backend issue) but I figured I'd at least fix the frontend problem.